package.json

```diff
@@ -4,19 +4,20 @@
   "description": "",
   "main": "lib/index.js",
   "scripts": {
-    "build": "rimraf ./lib && tsc && ncp ./src/client/proto/ ./lib/client/proto/",
-    "prepublish": "npm run test && npm run build",
+    "build": "rimraf ./lib && tsc && ncp ./src/client/ ./lib/client/",
+    "prepublish": "./generate && npm run test && npm run build",
     "test": "ts-node ./node_modules/tape/bin/tape 'src/**/*_test.ts'",
-    "coverage": "rimraf ./istanbul && tsc --project tsconfig-istanbul.json && ncp ./src/client/proto/ ./istanbul/client/proto/ && istanbul cover -x 'istanbul/**/*_test.js' --report html ./node_modules/.bin/tape 'istanbul/**/*_test.js'",
-    "dev": "nodemon -e ts src --exec 'npm run test'"
+    "coverage": "rimraf ./istanbul && tsc --project tsconfig-istanbul.json && ncp ./src/client/ ./istanbul/client/ && istanbul cover -x 'istanbul/**/*_test.js' --report html ./node_modules/.bin/tape 'istanbul/**/*_test.js'",
+    "dev": "./generate && nodemon -e ts src --exec 'npm run test'"
   },
   "author": "Anthony Estebe <anthony@mesg.tech>",
   "license": "ISC",
   "dependencies": {
     "@grpc/proto-loader": "^0.3.0",
```

Comment:
I guess we can remove "@grpc/proto-loader"

Reply:
done